### ISSUE TITLE
Deprecate this module?

### ISSUE THREAD

**Issue by marc-farre** [humhub-contrib/termsbox#12]
@luke- Shouldn't we deprecate this module as we already have the "Legal Tools" module which is better?
I think it would avoid confusions for people which doesn't know there are 2 modules for legal terms.

**Comment by marc-farre** [comment]
Moreover, it seems this module has conflict with the "Legal Tools" (not tested): https://community.humhub.com/comment/perma?id=49451

**Comment by luke-** [comment]
@marc-farre Good point, just marked it as deprecated in the marketplace. And I'll archive this repo,
